charts/jx3/jx-pipelines-visualizer/defaults.yaml

```diff
@@ -1,2 +1,2 @@
 gitUrl: https://github.com/jenkins-x/jx-pipelines-visualizer
-version: 0.0.52
+version: 0.0.53
```

Comment:
assuming https://github.com/jenkins-x/jx-pipelines-visualizer/pull/41 gets merged and creates new release